R/ResampleInstances.R

```diff
@@ -38,3 +38,45 @@ instantiateResampleInstance.RepCVDesc = function(desc, size, task = NULL) {
   g = as.factor(rep(seq_len(desc$reps), each = folds))
   makeResampleInstanceInternal(desc, size, train.inds = train.inds, test.inds = test.inds, group = g)
 }
+
+instantiateResampleInstance.FixedCVDesc = function(desc, size, task = NULL) {
+  initial.window = floor(desc$initial.window * size)
+  skip = floor(desc$skip * size)
+  stops  = (seq(size))[initial.window:(size - desc$horizon)]
+  starts = stops - initial.window + 1
+  train.inds = mapply(seq, starts, stops, SIMPLIFY = FALSE)
+  test.inds  = mapply(seq, stops + 1, stops + desc$horizon, SIMPLIFY = FALSE)
+
+  thin = function(x, skip = 2) {
+    n = length(x)
+    x[seq(1, n, by = skip)]
+  }
+
+  if (skip > 0) {
+    train.inds = thin(train.inds, skip = skip + 1)
+    test.inds = thin(test.inds, skip = skip + 1)
+  }
+  desc$iters = length(test.inds)
+
+  makeResampleInstanceInternal(desc, size, train.inds = train.inds, test.inds = test.inds )
+}
+
+instantiateResampleInstance.GrowingCVDesc = function(desc, size, task = NULL) {
```

Comment:
All my comments to `instantiateResampleInstance.FixedCV` apply in this function as well.